### DISCUSSION TITLE
怎么把出图检测禁了

### DISCUSSION THREAD

**Discussion by VinkleLynx** [Melledy/Grasscutter#1767] [Q&A]
想去看看赤王陵 刚出去直接前面的蛆 血压一下就高起来了 旧版明明没有这弱智功能的

**Comment by ChisatoNishikigi73** [comment]
删除PlayerProgressManager.java中关于BLACKLIST_OPEN_STATES的代码以跳过地图边界的黑名单。

**Reply by ChisatoNishikigi73** [reply]
因为要完善任务解锁，Remove this as soon as quest unlocks are fully implemented.

**Comment by Hartie95** [comment]
Its possible to manually disable the border via setProp command:
`/setProp setopenstate 48`

**Comment by Jimmy32767255** [comment]
> 想去看看赤王陵 刚出去直接前面的蛆 血压一下就高起来了 旧版明明没有这弱智功能的

请注意言辞，更像官方服务器和更完整一直是此项目的目标，这并不是什么弱智功能
